dts/arc/tenstorrent/tt_blackhole_fixed_partitions.dtsi

```diff
@@ -15,55 +15,55 @@
 
 		cmfwcfg@14000 {
 			label = "cmfwcfg";
-			reg = <0x014000 0x50>;
+			reg = <0x014000 0x100>;
 			binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/fw_table.bin";
 			read-only;
 		};
 
 		origcfg@15000 {
 			label = "origcfg";
-			reg = <0x015000 0x50>;
+			reg = <0x015000 0x100>;
 			binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/fw_table.bin";
 			read-only;
 		};
 
 		cmfw@16000 {
 			label = "cmfw";
-			reg = <0x016000 0x19bb0>;
+			reg = <0x016000 0x19fff>;
```

Comment:
Why not round this to 0x1a000?

Reply:
I'm not sure if the continuous memory region is causing a problem, so I want to leave some gap

Reply:
What were the failures?

Reply:
It was the same ones I'm getting now